dns_server/src/crawler/storage_impl.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::error::DnsServerError;
+
+use super::storage::{
+    DnsServerStorage, DnsServerStorageRead, DnsServerStorageWrite, DnsServerTransactionRo,
+    DnsServerTransactionRw, DnsServerTransactional,
+};
+
+storage::decl_schema! {
+    /// Database schema for peer db storage
+    pub Schema {
+        /// Table for all reachable addresses
+        pub DBAddresses: Map<String, ()>,
```

Comment:
Please add a schema for versioning too. You need to store the version of your schema.

Reply:
Sure, added.